ops/docker/op-stack-go/Dockerfile

```diff
@@ -108,8 +86,14 @@ RUN curl -L https://github.com/casey/just/releases/download/$(yq '.tools.just' m
 # Go sources
 COPY ./go.mod /app/go.mod
 COPY ./go.sum /app/go.sum
-# Copy rust libs for dynamic linking
-COPY --from=rust-builder /libespresso/* /lib
+# Fetch rust libs for dynamic linking
+ARG ESPRESSO_NETWORK_GO_VER=0.3.2
+ADD --checksum=sha256:ec6ce7b37edd173206ad338c84a6a771a0e9dc8b184081af7440ebfc0c531a71 \
```

Comment:
We have an argument to swap the version of the library we're downloading, but the checksum is static and not an argument.  This contradicts itself.  At the very least we should probably add an argument for the checksum as well.